test/e2e/main.go

```diff
@@ -56,7 +56,7 @@ var (
 	ginkgoFocus         = flag.String("ginkgo-focus", "", "pass to ginkgo run --focus flag")
 	ginkgoSkip          = flag.String("ginkgo-skip", "", "pass to ginkgo run --skip flag")
 	ginkgoProcs         = flag.String("ginkgo-procs", "10", "pass to ginkgo run --procs flag")
-	ginkgoTimeout       = flag.String("ginkgo-timeout", "4h", "pass to ginkgo run --timeout flag")
+	ginkgoTimeout       = flag.String("ginkgo-timeout", "8h", "pass to ginkgo run --timeout flag")
```

Comment:
please make a note to check the timeout, and the frequency of test runs for the internal prow test config